src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using SonarLint.VisualStudio.Integration.Vsix;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    [TestClass]
+    public class SonarLintDaemonTests
+    {
+        private string tempPath;
+        private string storagePath;
+        private SonarLintDaemon daemon;
+
+        [TestInitialize]
```

Comment:
I guess you already know it, but the test framework creates a new instance of the test class (e.g. SonarLintDaemonTests) for each test it is executing. Then it calls the TestInitialize (e.g. SetUp) method... It is definitely not a problem here, but don't feel tempted to share state between tests :)